q2_viromics/_utils.py

```diff
@@ -0,0 +1,123 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2024, Bokulich Lab.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+import os
+import subprocess
+from typing import List
+
+EXTERNAL_CMD_WARNING = (
+    "Running external command line application(s). "
+    "This may print messages to stdout and/or stderr.\n"
+    "The command(s) being run are below. These commands "
+    "cannot be manually re-run as they will depend on "
+    "temporary files that no longer exist."
+)
+
+
+def get_full_path(file_name):
+    """
+    Convert a file name or a relative path to its absolute path.
+
+    Args:
+    file_name (str): The name of the file or the relative path to the file.
+
+    Returns:
+    str: The absolute path of the file.
+    """
+
+    return os.path.abspath(file_name)
+
+
+def create_directory(path):
+    """
+    Create a directory at the specified path. If the directory already exists,
+    no action is taken.
+
+    Args:
+    path (str): The path where the directory is to be created.
+
+    Returns:
+    bool: True if the directory was created, False if it already existed.
+    """
+
+    try:
+        os.makedirs(path)
+        return True
+    except FileExistsError:
+        return False
+
```

Comment:
Same here why do you have this? You are not using it.
this is also the same as just using `os.makedirs(path, exists_ok: True)`